app/views/content/welcome/landing/_exciting-times-ahead.html.erb

```diff
@@ -12,47 +12,79 @@
     <div class="stage">
       <div class="card">
         <div class="number">1</div>
-        <%= image_pack_tag("media/images/content/welcome-guide/school.svg") %>
-        <div class="stage-name">Get experience.</div>
+        <%= image_pack_tag("media/images/content/welcome-guide/school.svg", alt: "") %>
```

Comment:
Change intro text in row 6/7 to:

Look out for our emails over the next few weeks. They're packed with advice and support, as well as more inspiring teacher stories. In the meantime, our website has lots of great content to help you as you consider a career in teaching.